tests/lax_numpy_test.py

```diff
@@ -1351,6 +1351,54 @@ def testPad(self, shape, dtype, mode, pad_width, constant_values, rng_factory):
                             check_dtypes=shape is not jtu.PYTHON_SCALAR_SHAPE)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape={}_mode={}_pad_width={}_stat_length={}".format(
+          jtu.format_shape_dtype_string(shape, dtype), mode, pad_width, stat_length),
+       "shape": shape, "dtype": dtype, "mode": mode, "pad_width": pad_width,
+       "stat_length": stat_length, "rng_factory": jtu.rand_default}
+      for mode, shapes in [
+          ('maximum', nonempty_shapes),
+          ('minimum', nonempty_shapes),
+          ('mean', nonempty_shapes),
+          ('median', nonempty_shapes),
+      ]
+      for shape, dtype in _shape_and_dtypes(shapes, all_dtypes)
+      for pad_width in [
+          # ((before_1, after_1), ..., (before_N, after_N))
+          tuple((i % 3, (i + 1) % 3) for i in range(len(shape))),
+          # ((before, after),)
+          ((1, 2),), ((2, 0),),
+          # (before, after)  (not in the docstring but works in numpy)
+          (2, 0), (0, 0),
+          # (pad,)
+          (1,), (2,),
+          # pad
+          0, 1,
+      ]
+      for stat_length in [
+          # ((before_1, after_1), ..., (before_N, after_N))
+          tuple(((i % 3 + 1), ((i + 1) % 3) + 1) for i in range(len(shape))),
+          # ((before, after),)
+          ((1, 2),), ((2, 2),),
+          # (before, after)  (not in the docstring but works in numpy)
+          (1, 1), (3, 4),
+          # (pad,)
+          (1,), (2,),
+          # pad
+          1, 2
+      ]
+      if pad_width != ()))
+  def testPadStatValues(self, shape, dtype, mode, pad_width, stat_length, rng_factory):
+    rng = rng_factory(self.rng())
```

Comment:
Minor: recently, we've started using `jtu.rand_default()` directly to avoid having to add `rng_factory` to all the parametrization boilerplate.